### ISSUE TITLE
while sign-up, the link for 'FightPandemics' on 'Check your inbox' page is not working 

### ISSUE THREAD

**Issue by sheeba1** [FightPandemics/FightPandemics#1249]
Issue: while sign-up, the link for 'FightPandemics' on 'Check your inbox' page is not working . Refer picture below:

steps:1. Go to  https://rcv11.fightpandemics.xyz/ , Navigate to Join Now

2.Enter email,password and confirm password and click 'Join Now'
3.'Check your Inbox' page is displayed
4. click on FP link on top left
Issue: FP link is not working

![image](https://user-images.githubusercontent.com/65972693/88022253-ccc3ea80-cae3-11ea-90fc-dbbdb48c973b.png)

**Comment by joshmorel** [comment]
Thanks @sheeba1 . 

This is behaving as such for similar reasons explained in #1153. It wouldn't be easy to change (because of multiple functions depended on the same redirect logic) without refactoring to auth (which I'm working on slowly).

I'm thinking the best solution now and maybe even later is for these auth pages:

* create-profile
* auth/signup
* auth/login
* auth/check-email
* auth/forgot-password

We should just make the logo non-clickable so there's confusion. And for some of the pages we don't want them to navigate away (although they still can be clicking back button).

Thoughts @indirabhowmick @vikjoshi @manualzuru ?

FYI these pages are the ones with navless headers like:

![image](https://user-images.githubusercontent.com/10546720/88095301-e22c2980-cb62-11ea-8e32-502340c55c08.png)

**Comment by sheeba1** [comment]
Comment from Manu, "@Sheeba Jose in most of the transactional features we don't want them to click anywhere else. Please check Amazon checkout page"

So we can make logo unclickable?

**Comment by joshmorel** [comment]
@sheeba1  Yeah that was my thought. It would be fairly easy.

Right now it's wrapped in a `<Link>` like such:

```jsx
          <Link to="/">
            <Logo src={logo} alt="Fight Pandemics logo" />
          </Link>
```

Where we don't want it clickable we just omit wrapping it in a `<Link>`.

If this is a go then either we can reword this issue or close this and create a new one.

**Comment by manualzuru** [comment]
@joshmorel yeah you are correct. In transactional pages (Login/Signup) we want them to not be able to click on the logo so they don't leave and complete. This is a normal behavior, check Amazon checkout page.
